README.md

```diff
@@ -33,14 +33,21 @@
     * For the food-truck-finder database, right click and navigate to New > Database
     * Add a new database named food-truck-finder and hit OK
 
-* Startup the API from IntelliJ SpringBoot Run Configuration
+* Start the API from IntelliJ SpringBoot Run Configuration
     * Specify VM Options
 	```
-	-Dspring.profiles.active=development
+	-Dspring.profiles.active=dev
 	```
+    * Specify Environment Variables
     ```
-    GET localhost:8080/ping
-    pong!
+    GOOGLE_SMTP_USERNAME=your.smtp.email@gmail.com
+    GOOGLE_SMTP_PASSWORD=your_smtp_password
+    SIMILARITY_URL=http://localhost:5000
```

Comment:
Optional variables that we may wish to note:
`MIN_FOR_EXACT`: Minimum similarity score needed to conclude two tags are identical. Defaults to 0.98
`MIN_FOR_TYPE`: Minimum similarity score needed to conclude two tags reside in the same food group. Defaults to 0.65 (based off of the similarity score for 'taco' and 'chimichanga'--kind of arbitrary, but it seemed to make sense)